src/lib/storage/table.hpp

```diff
@@ -19,36 +20,59 @@ namespace opossum {
 
 class TableStatistics;
 
-// A table is partitioned horizontally into a number of chunks
+/**
+ * A Table is partitioned horizontally into a number of chunks.
+ *
+ *
```

Comment:
A few too many comment lines for my taste.

Reply:
you think so? tsss....